evergreen/remove-certs.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+if [[ "$OS" =~ Windows|windows ]]; then
+    certutil.exe -delstore "Root" `openssl x509 -noout -text -in ${DRIVERS_TOOLS}/.evergreen/x509gen/ca.pem\
+    | grep Serial\
+    | sed 's/.*(0x\(.*\))/\1/'`
+fi
```

Comment:
`openssl x509 -noout -text -in ${DRIVERS_TOOLS}/.evergreen/x509gen/ca.pem`
yields
```
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 480006 (0x75306)
[snip]
```
So we' `grep` for the line with the Serial Number, and then use `sed` to extract the hex representation. Finally we pass the cert's serial number to `certutil.exe` so that it can remove it from the "Root" store.